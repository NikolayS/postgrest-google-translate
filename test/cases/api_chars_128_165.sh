#!/bin/bash

psql -U postgres test -c "select v1.google_translate('ru', 'en', 'test: ' || string_agg(char, '')) from (select chr(generate_series) as char from generate_series(127, 165)) as z"
