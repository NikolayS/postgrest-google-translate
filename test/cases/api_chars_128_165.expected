 google_translate 
------------------
 test:
(1 row)

